op-e2e/actions/kromampt_fork_test.go

```diff
@@ -0,0 +1,182 @@
+	for i := 1; i < 12; i++ {
+		txn := transactions[i]
+		receipt, err := ethCl.TransactionReceipt(context.Background(), txn.Hash())
+		require.NoError(t, err)
+		require.Equal(t, types.ReceiptStatusSuccessful, receipt.Status)
+		require.NotEmpty(t, txn.Data(), "upgrade tx must provide input data")
+	}
+
+	expectedL1BlockAddress := crypto.CreateAddress(derive.L1BlockMPTDeployerAddress, 0)
+	expectedBaseFeeVaultAddress := crypto.CreateAddress(derive.BaseFeeVaultDeployerAddress, 0)
+	expectedL1FeeVaultAddress := crypto.CreateAddress(derive.L1FeeVaultDeployerAddress, 0)
+	expectedSequencerFeeVaultAddress := crypto.CreateAddress(derive.SequencerFeeVaultDeployerAddress, 0)
+	expectedGasPriceOracleAddress := crypto.CreateAddress(derive.GasPriceOracleMPTDeployerAddress, 0)
+
+	// New Base Fee Vault impl is deployed
+	updatedBaseFeeVaultAddress, err := ethCl.StorageAt(context.Background(), oppredeploys.BaseFeeVaultAddr, genesis.ImplementationSlot, latestBlock.Number())
+	require.NoError(t, err)
+	require.Equal(t, expectedBaseFeeVaultAddress, common.BytesToAddress(updatedBaseFeeVaultAddress))
+	verifyCodeHashMatches(t, ethCl, expectedBaseFeeVaultAddress, baseFeeVaultMPTCodeHash)
+
+	// New L1 Fee Vault impl is deployed
+	updatedL1FeeVaultAddress, err := ethCl.StorageAt(context.Background(), oppredeploys.L1FeeVaultAddr, genesis.ImplementationSlot, latestBlock.Number())
+	require.NoError(t, err)
+	require.Equal(t, expectedL1FeeVaultAddress, common.BytesToAddress(updatedL1FeeVaultAddress))
+	verifyCodeHashMatches(t, ethCl, expectedL1FeeVaultAddress, l1FeeVaultMPTCodeHash)
+
+	// New Sequencer Fee Vault impl is deployed
+	updatedSequencerFeeVaultAddress, err := ethCl.StorageAt(context.Background(), oppredeploys.SequencerFeeVaultAddr, genesis.ImplementationSlot, latestBlock.Number())
+	require.NoError(t, err)
+	require.Equal(t, expectedSequencerFeeVaultAddress, common.BytesToAddress(updatedSequencerFeeVaultAddress))
+	verifyCodeHashMatches(t, ethCl, expectedSequencerFeeVaultAddress, sequencerFeeVaultMPTCodeHash)
+
+	// Gas Price Oracle impl is updated
+	updatedGasPriceOracleAddress, err := ethCl.StorageAt(context.Background(), predeploys.GasPriceOracleAddr, genesis.ImplementationSlot, latestBlock.Number())
+	require.NoError(t, err)
+	require.Equal(t, expectedGasPriceOracleAddress, common.BytesToAddress(updatedGasPriceOracleAddress))
+	require.NotEqualf(t, initialGasPriceOracleAddress, updatedGasPriceOracleAddress, "Gas Price Oracle Proxy address should have changed")
+	verifyCodeHashMatches(t, ethCl, expectedGasPriceOracleAddress, gasPriceOracleMPTCodeHash)
+
+	// New L1Block impl is deployed
+	updatedL1BlockAddress, err := ethCl.StorageAt(context.Background(), oppredeploys.L1BlockAddr, genesis.ImplementationSlot, latestBlock.Number())
+	require.NoError(t, err)
+	require.Equal(t, expectedL1BlockAddress, common.BytesToAddress(updatedL1BlockAddress))
+	require.NotEqualf(t, initialL1BlockAddress, updatedL1BlockAddress, "L1Block Proxy address should have changed")
```

Comment:
We don't need this line cause we deployed L1Block contract to new address.

Reply:
Oh I meant to just remove the address comparison part between initialL1BlockAddress and updatedL1BlockAddress (only L148)